Sources/StreamChatUI/Utils/ImageLoading/ImageLoading.swift

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+/// Cancellable provides a set of functions that enable cancelling a task
+public protocol Cancellable {
```

Comment:
should be moved in separate file `Cancellable.swift`

Reply:
cause its very generic, and can be used out of ImageLoading proto

Reply:
Makes sense, moved it to a separate file